test/extensions/filters/http/fault/fault_filter_integration_test.cc

```diff
@@ -142,6 +144,7 @@ TEST_P(FaultIntegrationTestAllProtocols, HeaderFaultConfig) {
   EXPECT_EQ(0UL, test_server_->counter("http.config_test.fault.aborts_injected")->value());
   EXPECT_EQ(1UL, test_server_->counter("http.config_test.fault.delays_injected")->value());
   EXPECT_EQ(1UL, test_server_->counter("http.config_test.fault.response_rl_injected")->value());
+  EXPECT_EQ(0UL, test_server_->gauge("http.config_test.fault.active_faults")->value());
```

Comment:
Can/should we verify on line 138 that `http.config_test.fault.active_faults == 1`?

Reply:
Yes, that’s a good idea - I’m going to improve these tests in a separate PR.

Reply:
Nvm, just pushed my changes with suggested improvements